### DISCUSSION TITLE
Backup/Restore documentation outdated?

### DISCUSSION THREAD

**Discussion by ssteiner** [ravendb/ravendb#19045] [General]
I'm currently looking at the backup/restore functionality. So far I'm using a free developer edition, so I expect my options to be somewhat limited - but I'm quite confident that the documentation wasn't written for the current release. I'm looking at this page in particular: [Cloud backup and restore](https://ravendb.net/docs/article-page/6.0/csharp/cloud/cloud-backup-and-restore). It has the 6.0 version in the URL, so should be current, right?

The first picture showing the where you see and configure the tasks. In the Studio, in the settings for my database, I have no Tasks section at the bottom. In fact, in my studio, in the menu on the left hand side, I have Documents (blue, with a documents icon), Indexes (violet - with an Icon I can't describe)), Tasks (red, with a clipboard icon), Configuration (yellow, with a toothwheel icon) and stats (green, with a bar chart icon). The most fitting seems to be Tasks, which has an 'Ongoing Tasks' option which seems to be it. And no chance to find this based on the picture in the documentation (or the steps described in the documentation). In fact, the documentation for the latest studio should say

1) Login to studio
2) Go to Tasks
3) Click on Ongoing Tasks

That matches what I'm seeing. 

Instead the documentation states:

> Choose the database whose backup task you want to view
> Click Manage Ongoing Tasks
> Click the "Server Wide Backup" task.

Took me a couple of minutes to find that. And then scrolling down in the documentation, the other pictures also seem to be from an older release - it's rather hard to follow instructions that refer to a GUI that no longer looks like that. Please consider updating the documentation if things are moved around in the GUI.

**Comment by Danielle9897** [comment]
Created an issue to handle this:
https://issues.hibernatingrhinos.com/issue/RDoc-2965/Cloud-backup-Restore-Update-article
